modules/openapi/tests/src/StructureSpec.scala

```diff
@@ -65,6 +65,41 @@ final class StructureSpec extends munit.FunSuite {
     TestUtils.runConversionTest(openapiString, expectedString)
   }
 
+  test("structures - inline uuid example") {
+    val openapiString = """|openapi: '3.0.'
+                     |info:
+                     |  title: test
+                     |  version: '1.0'
+                     |paths: {}
+                     |components:
+                     |  schemas:
+                     |    Object:
+                     |      type: object
+                     |      properties:
+                     |        u:
+                     |          format: uuid
+                     |          type: string
+                     |          example: 630fe836-9220-11e7-abc4-cec278b6b50a
```

Comment:
this was the way to reproduce the issue, other tests modifications did not trigger an issue

only an example inlined in the property definition did